TEST_AUTOMATION.md

```diff
@@ -0,0 +1,129 @@
+# ACPaaS UI Test Automation Guide
+
+When automating tests, one of the challenges is to keep the test resilient to code changes.
+Using [data-attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) can help u with that, by creating unique selectors for your tests.
+
+## Using uniqe data-attributes with ACPaaS UI
+
+It's not possible to add uniqe data-attributes to the ACPaaS UI component library for the simple reason that the attribute won't be unique anymore when u use the component multiple times on the same page.
+
+But in your app you can simply add a data-attribute to each individual ACPaaS UI component so you can target the component that you want to test.
```

Comment:
Two ways to spell 'data attributes' correctly:
1. the one I just used
2. data-* attributes